### DISCUSSION TITLE
GitHub Actions (CI/CD) - Nothing happens

### DISCUSSION THREAD

**Discussion by gulien** [community/community#25896] [Actions]
<p>Hello,</p>
<p>I’m currently playing with GitHub Actions v2 (aka CI/CD) in this repository: <a href="https://github.com/thecodingmachine/github-actions-test/" rel="nofollow noopener">https://github.com/thecodingmachine/github-actions-test/</a></p>
<p>I’ve setup my workflows inside <code>.github/workflows</code> and I expect the tab <code>Actions</code> to display them.</p>
<p>Alas, nothing is displayed!</p>
<p>Also, no workflows are triggered, whenever I push, open a pull request or create a tag.</p>
<p>What did I miss?</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/gulien">@gulien</a>,</p>
<p>Thanks for being here, looks like you’ve not been flagged into the new version yet. We are getting more folks into the newest version daily.</p>

**Comment by gulien** [comment]
<p>Hello Andrea!</p>
<p>Actually the organization <span class="mention">@thecodingmachine</span> should be flagged into the new version as I am able to use Github Actions v2 in another repository :smile:</p>
<p>Or maybe not all repositories are flagged into the new version?</p>
<p>Thank you</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hmm interesting, let me take a look and see what’s going on</p>

**Comment by tobiaskohlbau** [comment]
<p>I can confirm that I noticed the same problem. Some repositorys of the same org(which is enabled for the new actions beta) stick to the HCL actions and don’t offer the “upgrade/update” button others do.</p>

**Comment by gulien** [comment]
<p>Hello <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> </p>
<p>Any news? :slight_smile:</p>
<p>Thank you,</p>
<p>Julien</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/gulien">@gulien</a>,</p>
<p>Thank you for being here and again for your patience while we investigated, we’ve found the issue to be that although your organization had been flagged in, that particular repo was not. I’ve updated this and it should now be v2 enabled. Please give it a go and feel free to let me know if you have any other questions.</p>
<p>Thanks again for your patience,</p>
<p>ps: <a class="mention" href="/tobiaskohlbau">@tobiaskohlbau</a> would you mind opening a new post here and linking your repo’s so we can take a look.</p>

**Comment by gulien** [comment]
<p>Thank you very much!</p>

**Comment by tobiaskohlbau** [comment]
<p><a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> I wouldn’t post the repository in public as it’s a private repo. Are there any other options to reach out?</p>
